docs/content/community/presentations/index.md

```diff
@@ -0,0 +1,25 @@
+---
+type: docs
+title: "Give a Radius presentation"
+linkTitle: "Presentations"
+description: "Learn how to give a Radius presentation, with downloadable slides and assets"
+weight: 400
+---
+
+We welcome community members giving presentations on Radius and spreading the word about all the awesome Radius features!
+
+## Giving a Radius presentation
+
+1. Begin by downloading the Radius Presentation Deck. This contains slides and diagrams needed to give a Radius presentation.
+1. Next, review the docs to make sure you understand the concepts.
+1. Use the Radius [getting started guide]({{< ref getting-started >}}) and [samples repo](https://github.com/radius-project/samples) to show demos of how to use Dapr.
```

Comment:
Not sure if this is a typo or if it was meant to show a combination of dapr + radius :)

```suggestion
1. Use the Radius [getting started guide]({{< ref getting-started >}}) and [samples repo](https://github.com/radius-project/samples) to show demos of how to use Radius.
```